tests/test_calculation.py

```diff
@@ -1,183 +0,0 @@
-"""Test for running calculations on a FireCREST computer."""
```

Comment:
Why is this file removed?

Reply:
Shouldn't have :)) 
I'll adopt it right away